### PR TITLE
Hrinfo 45

### PR DESCRIPTION
**Pull request by humphreymburu**
Hi Guys 

In regards to this case I made two commits:
 1.On August 28 -  related to day view display
 2.September 22  - related to week view display

Note :The first commit I git added scss.map file by mistake which I should not apart from that it should be okay.

@pcambra and @MartinMaritim review the changes so we can close this case.

Cheers

### CODE REVIEW

File: sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss [hunk 1]
```diff
@@ -632,6 +881,7 @@ a.fc-event, .fc-event-draggable {
 
 		div.single-day .view-item {
 			width: 50%;
+			width: 200px;
```
Comment (pcambra) [inline sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss]: Any reason for having two widths?

File: sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+          <?php endforeach; ?>   
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+<div class="header-body-divider">&nbsp;</div>
+<div id="single-day-container">
+  <?php if (!empty($scroll_content)) : ?>
+  <script>
```
Comment (pcambra) [inline sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php]: We shouldn't have this, it should be a behavior instead

File: sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  <?php endif; ?>
+  <table class="full">
+    <tbody>
+      <tr class="holder">
+        <td class="calendar-time-holder"></td>
+        <td class="calendar-day-holder"></td>
+      </tr>
+      <tr>
+        <td class="first">
+          <?php $is_first = TRUE; ?>
```
Comment (pcambra) [inline sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php]: Can we do this in a preprocess?

File: sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+<div class="single-day-footer">&nbsp;</div>
+</div></div>
+<?php if (!empty($scroll_content)) : ?>
+<script>
+try {
```
Comment (pcambra) [inline sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php]: More Js that should go into a separate file

File: sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss (hunk 1 above)
Comment (humphreymburu) [inline sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss]: I might have missed that width.

File: sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php (hunk 2 above)
Comment (humphreymburu) [inline sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php]: I used the day-overlap tpl.php code that comes with calendar module.

File: sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php (hunk 3 above)
Comment (humphreymburu) [inline sites/all/themes/humanitarianresponse/templates/view/calendar-day-overlap--hr-events-calendar.tpl.php]: Using a preprocess is quite complicated and lacks resources to solve this case.

**Comment by humphreymburu** [comment]
Hi @pcambra what if I can solve the tpl.php issues with CSS only ?

**Comment by humphreymburu** [comment]
@pcambra I removed the calendar-day-overlap--hr-events-calendar.tpl.php file , the tpl's goal has been replaced by css option.

Check if the implementation of day/week view is okay.

Cheers

**Comment by pcambra** [comment]
I still see the tpl there, also, please make sure that every commit is related with the issue.

**Comment by humphreymburu** [comment]
@pcambra last time I forgot to rm so used to adding ,anyway check now.

**Comment by pcambra** [comment]
Could you please fix the commits that don't link to the issue? you can use git rebase for it, let me know if you need help

File: sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss [hunk 5]
```diff
@@ -343,6 +434,168 @@ a.fc-event, .fc-event-draggable {
 	}
 
 	.calendar-calendar .week-view td:before {
+		display: inline-block;
+	}
+
+	.calendar-calendar .week-view .full .days {
+       width: 13%;
```
Comment (pcambra) [inline sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss]: There's a mixture of line spacing and tabs here

File: sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss [hunk 6]
```diff
@@ -343,6 +434,168 @@ a.fc-event, .fc-event-draggable {
...
+    }
+
+	.calendar-calendar .week-view #single-day-container table td {
+		padding: 0px;
+		border-top: 0px;
+		border-bottom: 0px;
+		width: 10%;
+		display: inline-block;
+		background-color: white;
+		border: #ffffff;
```
Comment (pcambra) [inline sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss]: You're using "white" and #ffff that mean the same thing

File: sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss [hunk 7]
```diff
@@ -343,6 +434,168 @@ a.fc-event, .fc-event-draggable {
...
+		float: right;
+		position: relative;
+		clear: both;
+		width: 130px;
+		left: 25px;
+		font-weight: bold;
+	}
+
+	.calendar-calendar .week-view .calendar-agenda-items.multi-day.no-entry.last {
+		display: none;
```
Comment (pcambra) [inline sites/all/themes/humanitarianresponse/assets/sass/partials/_view.scss]: you can separate by commas and do just one display none

**Comment by humphreymburu** [comment]
@pcambra I have rectified the css changes you highlighted.

**Pull request merged**